lib/mpl_toolkits/axisartist/axis_artist.py

```diff
@@ -230,27 +230,19 @@ def get_markeredgewidth(self):
         return self.get_attribute_from_ref_artist("markeredgewidth", .5)
 
     def set_tick_out(self, b):
-        """
-        set True if tick need to be rotated by 180 degree.
-        """
+        """Set whether the ticks need to be rotated by 180 degree."""
```

Comment:
degreeS (below as well)
or just "whether ticks are drawn inside or outside the axes" which is clearer